ui/src/components/verifier/Verifier.tsx

```diff
@@ -107,12 +112,24 @@ const Verifier: React.FC = () => {
     }
 
     const checkResultToElement = (checkResult: ServersideAddressCheck) => {
-        return <p key={checkResult.address}>{checkResult.address} is verified on: {
-            intersperse(
-                checkResult.chainIds.map(chainId => chainToLink(chainId, checkResult.address)), 
-                ", "
-            )
-        }</p>
+        return (
+            <>
+                <p key={checkResult.address}>{checkResult.address} is fully verified on: {
+                    intersperse(
+                        checkResult.chainIds.filter(chain => chain.status === verificationState.PERFECT).map(chainId => chainToLink(chainId.chainId, checkResult.address)), 
+                        ", "
+                    )
+                }
+                </p>
+                <p key={checkResult.address}>Also partially verified on: {
+                    intersperse(
+                        checkResult.chainIds.filter(chain => chain.status === verificationState.PARTIAL).map(chainId => chainToLink(chainId.chainId, checkResult.address)), 
```

Comment:
Here `chainToLink` seems to create a full match link to the repository. It should be `/contracts/partial_match/:chainId/:address`

Reply:
Thanks, good catch🙂